torch/utils/cpp_extension.py

```diff
@@ -195,7 +195,11 @@ def include_paths(cuda=False):
     lib_include = os.path.join(torch_path, 'lib', 'include')
     # Some internal (old) Torch headers don't properly prefix their includes,
```

Comment:
Can you post an issue to fix this? It doesn't seem that hard to do and would stop making all of our build flags awful.

Reply:
https://github.com/pytorch/pytorch/issues/5480